spec/models/course_spec.rb

```diff
@@ -333,4 +333,31 @@
       it { should_not include(course) }
     end
   end
```

Comment:
I don't see a unit test for `#enrichments`, which is a public method.

Reply:
added